src/menus/dto/menus.createMenu.dto.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { Types } from 'mongoose';
+
+export class CreateMenuDto {
+  image: string;
+
+  @ApiProperty()
+  name: string;
+
+  @ApiProperty()
+  description: string;
+
+  @ApiProperty()
+  price: number;
+
+  @ApiProperty()
+  @Transform(({ value }) => new Types.ObjectId(value))
+  category: string;
+
+  @ApiProperty()
+  @Transform(({ value }) => value.map((v) => new Types.ObjectId(v)))
+  addons: string[];
```

Comment:
When transform completed type will change to `Types.ObjectId`

Reply:
<img width="573" alt="image" src="https://github.com/meeorder/meeorder-backend/assets/35620424/a7cf983a-b164-444c-aa36-11376e04e135">

When I choose `Types.ObjectId` over `string` it's getting an error when use the endpoint.

Reply:
Will the frontend send it as an ObjectId type instead of a string?
Or it will convert to string anyway whether type that the frontend will send.

Reply:
Ok Solve